include/AdePT/core/AdePTTransport.h

```diff
@@ -41,9 +41,11 @@ class AdePTTransport : public AdePTTransportInterface {
   int GetNfromDevice() const { return fBuffer.fromDevice.size(); }
 
   /// @brief Adds a track to the buffer
-  void AddTrack(int pdg, int parentId, double energy, double x, double y, double z, double dirx, double diry,
-                double dirz, double globalTime, double localTime, double properTime, int threadId, unsigned int eventId,
-                unsigned int trackIndex, vecgeom::NavigationState &&state, vecgeom::NavigationState &&originState);
+  void AddTrack(int pdg, int parentId, double energy, double vertexEnergy, double x, double y, double z, double dirx,
```

Comment:
In future we may want to simplify this interface and at least pass Vector3D rather than components